desc/magnetic_fields.py

```diff
@@ -613,13 +613,20 @@ def compute_magnetic_field(self, coords, params=None, basis="rpz", grid=None):
             magnetic field at specified points
 
         """
+        if params is None:
```

Comment:
you should merge `rc/freeb` into this again so that the changes from  the PR that got merged into master already disappear from this one